darts/models/forecasting/baselines.py

```diff
@@ -44,6 +44,9 @@ def predict(self, n: int, num_samples: int = 1, verbose: bool = False):
         forecast = np.tile(self.mean_val, (1, n))
         return self._build_forecast_series(forecast)
 
+    def _is_probabilistic(self) -> bool:
```

Comment:
This is the default behvior of ForecastingModel._is_probabilistic() (these models are subclasses of ForecsatingModel).

We can remove this from here and from the other models below (non Ensemble models).